src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -7415,33 +7415,30 @@ private BoundExpression BindIndexedPropertyAccess(SyntaxNode syntax, BoundExpres
 
         private BoundExpression BindDynamicIndexer(
              SyntaxNode syntax,
-             BoundExpression receiverOpt,
+             BoundExpression receiver,
```

Comment:
We generally try not to change semantics of code in annotation PRs, especially in files that are not currently annotated.

Reply:
This is not a change in semantics.  There was no way for any caller to pass a null value before.  This just records that fact in the type system.

---
In reply to: [387953554](https://github.com/dotnet/roslyn/pull/42068#discussion_r387953554) [](ancestors = 387953554)